src/main/java/it/gov/pagopa/pu/workflow/wf/receiptclassification/iuf/dto/IufReceiptClassificationForTreasurySignalDTO.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.dto;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+@Builder
+public class IufReceiptClassificationForTreasurySignalDTO   {
```

Comment:
just call it `IufClassification`
```suggestion
public class IufClassificationNotifyTreasurySignalDTO   {
```